src/worker.ts

```diff
@@ -14,8 +26,28 @@ export default {
 			case EVERY_11_HOURS: {
 				return await addMissedUpdatesRole(env);
 			}
-			default:
+
+			case EVERY_20_MINUTES: {
+				await syncIdleUsers(env);
+				await syncNickNames(env);
+				await syncIdle7dUsers(env);
+				await syncOnboarding31dPlusUsers(env);
+				console.log('Worker for syncing idle users, nicknames, idle 7d users, and onboarding 31d+ users has completed.');
+				break;
+			}
+
+			case EVERY_30_MINUTES: {
+				await syncUsersStatus(env);
+				await syncExternalAccounts(env);
+				await syncUnverifiedUsers(env);
+				console.log('Worker for syncing user status, external accounts, and unverified users has completed.');
+				break;
```

Comment:
Console.log ?

Reply:
Just to help with debugging while tailing the logs